bin/i18n/sync-in.rb

```diff
@@ -586,6 +587,19 @@ def redact_level_content
   end
 end
 
+def redact_all_labs_content
+  puts "Redacting all *labs content"
+
+  Dir.glob(File.join(I18N_SOURCE_DIR, "blockly-mooc", "*lab.json")).each do |source_path|
```

Comment:
I would say get a second opinion, but it may be safer to create a constant array that holds all of the "labs" for now and then add/remove as necessary.

Reply:
I agree, i'm also moving in that direction in the Documentation Ticket. 
I decided to stick to whats included in the CSD Epic for now. Other labs can be added easily in future work.

Reply:
Array of labs implemented.